lib/ansible/plugins/callback/__init__.py

```diff
@@ -22,26 +22,18 @@
 import difflib
 import json
 import sys
-
+from collections import OrderedDict
```

Comment:
Nit: maybe keep the import section separator?
```suggestion

from collections import OrderedDict
```